index.js

```diff
@@ -5,19 +5,29 @@ import element from 'magic-virtual-element';
 let buttonIndex = 0;
 
 export function render ({ props }) {
-  const fn = 'youtubeSubscribeButtonEvent' + buttonIndex;
-  global[fn] = function (payload) {
-    if (payload.eventType === 'subscribe' && props.onSubscribe) {
-      props.onSubscribe();
-    }
-
-    if (payload.eventType === 'unsubscribe' && props.onUnsubscribe) {
-      props.onUnsubscribe();
-    }
-  };
-
-  const id = '__deku-yt-subscribe-wrapper-' + buttonIndex;
-  buttonIndex++;
+  let fn = '';
+  let id = '';
+
+  function setupCallbacks () {
+    fn = 'youtubeSubscribeButtonEvent' + buttonIndex;
+    global[fn] = function (payload) {
```

Comment:
since this is only getting used in the browser, why not set the params on the `window` within a namespace?